css/hotels-style.css

```diff
@@ -0,0 +1,201 @@
+@import url("https://fonts.googleapis.com/css?family=PT+Sans:400,400i,700,700i&display=swap&subset=cyrillic");
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: inherit;
+  box-sizing: inherit;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: 'PT Sans', Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 400;
+  color: #000;
+  text-transform: uppercase;
+  background-color: #fff;
+}
+
+.logo {
+  position: top center;
+}
+
+a {
```

Comment:
try to arrange selectors by tags higher than selectors by class cause selectors by class are global rules.
```
a {  
 text-decoration: none;
}
.logo {
  position: top center;
}
```